### DISCUSSION TITLE
2020-01-01

### DISCUSSION THREAD

**Discussion by giscus** [toshimaru/til.toshimaru.net#49] [Announcements]
# 2020-01-01

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

https://til.toshimaru.net/2020-01-01

**Comment by toshimaru** [comment]
test comment

**Comment by toshimaru** [comment]
test comment 2
